osi_common.proto

```diff
@@ -346,7 +345,7 @@ message BaseMoving
 
     // The relative orientation rate of the moving object w.r.t. its parent
     // frame and parent orientation rate in the center point of the bounding box
-    // (origin of the bounding box frame).
+    // (origin of the bounding box frame), noted in the parent frame.
```

Comment:
The orientation rate becomes global/absolute if the parent frame is not rotating.